cpp/dolfinx/nls/NewtonSolver.h

```diff
@@ -35,8 +35,20 @@ class NewtonSolver
   /// @param[in] comm The MPI communicator for the solver
   explicit NewtonSolver(MPI_Comm comm);
 
+  /// Move constructor (deleted)
```

Comment:
Don't document (`///`) deleted constructors.

Reply:
I think this is a limitation of our Doxygen settings (all methods must be documented.)